commands/new_function_test.go

```diff
@@ -0,0 +1,14 @@
+package commands
+
+//import "testing"
+//
+//func Test_new(t *testing.T) {
+//	//defer teardown()
```

Comment:
What are the commented files for?

Reply:
new_function_test: there is a function in the PR#87 which I need in this PR in order to clean up after the test, I'm waiting that PR to be merged before un-commenting this file

invoke_test: I haven't figured out how to copy data to os.Stdin inside the test, the test is not working, so I comment it out for now.

Reply:
Can you remove any commented code and keep in a separate development branch? If you would like help I'll see if I can look at this too - it'd be next week?